dcmdump/src/main.rs

```diff
@@ -54,6 +57,7 @@ where
             DumpValue::Str(v) => v.to_string().yellow(),
             DumpValue::DateTime(v) => v.to_string().green(),
             DumpValue::Invalid(v) => v.to_string().red(),
+            DumpValue::Nothing(v) => v.to_string().dimmed(),
```

Comment:
Either one would be OK (don't know of a way to color the second option without allocating though).

```suggestion
            DumpValue::Nothing => return Ok(()),
```

```suggestion
            DumpValue::Nothing => return write!("(no value)"),
```

Reply:
Would this work since it's implemented on `&str`? Still trying to get a handle on allocation and strings though

```rust
DumpValue::Nothing => "(no value)".dimmed(),
```